CefSharp.OutOfProcess.Interface/IOutOfProcessClientRpc.cs

```diff
@@ -35,8 +36,22 @@ public interface IOutOfProcessClientRpc
         /// <param name="parentHwnd">parent Hwnd</param>
         /// <param name="url">start url</param>
         /// <param name="browserId">browser id</param>
+        /// <param name="requestContextPreferences">Request Context Preferences to set.</param>
```

Comment:
The comment needs to be updated to describe the actual behaviour, a new `RequestContext` sharing the same settings as the `Global`.